gammapy/image/mask.py

```diff
@@ -95,8 +93,28 @@ def plot(self, ax=None, fig=None, **kwargs):
 
     @lazyproperty
     def distance_image(self):
-        """Map containting the distance to the nearest exclusion region."""
-        return exclusion_distance(self.mask)
+        """Distance to nearest exclusion region.
+
+        Compute distance map, i.e. the Euclidean (=Cartesian 2D)
+        distance (in pixels) to the nearest exclusion region.
+
+        We need to call distance_transform_edt twice because it only computes
+        dist for pixels outside exclusion regions, so to get the
+        distances for pixels inside we call it on the inverted mask
+        and then combine both distance images into one, using negative
+        distances (note the minus sign) for pixels inside exclusion regions.
+
+        Returns
+        -------
+        distance : `~numpy.ndarray`
```

Comment:
I don't care either. I think the distance image is mostly used internally in some algorithms. But it would be definitely useful to have the distance as an `Angle` or `Quantity` object with physical units attached.